### ISSUE TITLE
[Android] Unresolved reference: StreamSink

### ISSUE THREAD

**Issue by joaowd** [pycampers/flutter-rx-ble#17]
Trying to deploy the app gives me the following error:

```
e: /Users/joaopalma/Development/flutter/.pub-cache/hosted/pub.dartlang.org/rx_ble-0.5.5/android/src/main/kotlin/com/pycampers/rx_ble/ConnectMethods.kt: (3, 38): Unresolved reference: StreamSink
e: /Users/joaopalma/Development/flutter/.pub-cache/hosted/pub.dartlang.org/rx_ble-0.5.5/android/src/main/kotlin/com/pycampers/rx_ble/ConnectMethods.kt: (12, 52): Unresolved reference: StreamSink
e: /Users/joaopalma/Development/flutter/.pub-cache/hosted/pub.dartlang.org/rx_ble-0.5.5/android/src/main/kotlin/com/pycampers/rx_ble/ConnectMethods.kt: (16, 61): Unresolved reference: StreamSink
e: /Users/joaopalma/Development/flutter/.pub-cache/hosted/pub.dartlang.org/rx_ble-0.5.5/android/src/main/kotlin/com/pycampers/rx_ble/ReadWriteMethods.kt: (3, 38): Unresolved reference: StreamSink
e: /Users/joaopalma/Development/flutter/.pub-cache/hosted/pub.dartlang.org/rx_ble-0.5.5/android/src/main/kotlin/com/pycampers/rx_ble/ReadWriteMethods.kt: (17, 56): Unresolved reference: StreamSink
e: /Users/joaopalma/Development/flutter/.pub-cache/hosted/pub.dartlang.org/rx_ble-0.5.5/android/src/main/kotlin/com/pycampers/rx_ble/ReadWriteMethods.kt: (76, 65): Unresolved reference: StreamSink
e: /Users/joaopalma/Development/flutter/.pub-cache/hosted/pub.dartlang.org/rx_ble-0.5.5/android/src/main/kotlin/com/pycampers/rx_ble/ScanMethods.kt: (6, 38): Unresolved reference: StreamSink
e: /Users/joaopalma/Development/flutter/.pub-cache/hosted/pub.dartlang.org/rx_ble-0.5.5/android/src/main/kotlin/com/pycampers/rx_ble/ScanMethods.kt: (17, 49): Unresolved reference: StreamSink
e: /Users/joaopalma/Development/flutter/.pub-cache/hosted/pub.dartlang.org/rx_ble-0.5.5/android/src/main/kotlin/com/pycampers/rx_ble/ScanMethods.kt: (28, 58): Unresolved reference: StreamSink
```

Versions:
- rx_ble 0.5.5
- plugin_scaffold 1.1.1

This only happens in Android, iOS is fine.

**Comment by joaowd** [comment]
Forgot to say this only happens in Android

**Comment by TimmyGzw** [comment]
I also encountered this problem.

**Comment by devxpy** [comment]
Should be fixed in `v0.6.1`

**Comment by timmy-gzw** [comment]
available，Thank you！！！
